doc/lua_api.md

```diff
@@ -7532,17 +7532,33 @@ child will follow movement and rotation of that bone.
     object.
 * `set_detach()`: Detaches object. No-op if object was not attached.
 * `set_bone_position([bone, position, rotation])`
-    * `bone`: string. Default is `""`, the root bone
-    * `position`: `{x=num, y=num, z=num}`, relative, `default {x=0, y=0, z=0}`
-    * `rotation`: `{x=num, y=num, z=num}`, default `{x=0, y=0, z=0}`
-* `get_bone_position(bone)`:
-    * returns bone parameters previously set by `set_bone_position`
-    * returns `position, rotation` of the specified bone (as vectors)
-    * note: position is relative to the object
-* `set_properties(object property table)`:
-    * set a number of object properties in the given table
-    * only properties listed in the table will be changed
-    * see the 'Object properties' section for details
+	* Shorthand for `set_bone_override(bone, {position = position, rotation = rotation:apply(math.rad)})` using absolute values.
+	* Supports older servers & clients (5.8.0 and earlier).
```

Comment:
nitpick: this note seems to encourage using this function, however it is deprecated
that it supports old clients can also be deduced from below information

Reply:
Removed